src/Symfony/Component/Translation/Translatable.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation;
+
+use Symfony\Contracts\Translation\TranslatableInterface;
+
+/**
+ * @author Nate Wiebe <nate@northern.co>
+ */
+final class Translatable implements TranslatableInterface
+{
+    private $message;
+
+    private $parameters;
+
+    private $domain;
+
+    public function __construct(string $message, array $parameters = [], ?string $domain = null)
```

Comment:
the default domain is `messages`

Reply:
oh wait, null would mean "use the default domain from the translator". IMHO that's a bit too implicit; i'd prefer to bound message ID + domain in a single definition

Reply:
I think this makes sense, as long as the domain doesn't always have to be specified, as I would expect most simple projects to always use a single domain.